cardinal/server/persona.go

```diff
@@ -84,6 +85,19 @@ func (t *Handler) handleReadPersonaSignerSchema(w http.ResponseWriter, _ *http.R
 	writeResult(w, jsonSchema)
 }
 
```

Comment:
1generateCreatePersonaResponseFromPayload` takes some pure logic out of `makeCreatePersonaHandler` so that it can be re-used when creating the swagger handler logic for the swagger version of this same route.